### PR TITLE
174 add admin functionality

### PR DESCRIPTION
**Pull request by SvetoslavKazandzhiev**

### CODE REVIEW

File: Carwash/CW_places/urls.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 router = routers.DefaultRouter()
 router.register(r'cw_places', CWPlacesView)
 
+"""Define the mapping between URLs and views. """
 urlpatterns = [
-    path('', include(router.urls)),
+    path('api/', include(router.urls)),
```
Comment (strangerok1507) [inline Carwash/CW_places/urls.py:11]: what is the URL we receive for  GET list of CW places?

File: Carwash/CW_places/models.py [hunk 2]
```diff
@@ -4,11 +4,24 @@
 from places.models import Places
 
 
-class CWPlaces(models.Model):
-    carwash_place = models.IntegerField(primary_key=True, editable=False)
+class TimeStampedModel(models.Model):
```
Comment (strangerok1507) [inline Carwash/CW_places/models.py:7]: cool idea

File: Carwash/Carwash/settings.py [hunk 3]
```diff
@@ -146,3 +146,25 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     )
 }
+
+CELERY_BROKER_URL = "redis://redis:6379"
```
Comment (strangerok1507) [inline Carwash/Carwash/settings.py]: I recommend to use environment variables for CELERY_BROKER_URL and CELERY_RESULT_BACKEND

**Review by strangerok1507 — CHANGES_REQUESTED** [review]

**Comment by strangerok1507** [comment]
good work 👍 
I have a comments with minors changes

File: Carwash/CW_places/urls.py (hunk 1 above)
Comment (SvetoslavKazandzhiev) [reply Carwash/CW_places/urls.py:11]: This is the URL for GET - /api/cw_places/

File: Carwash/Carwash/settings.py (hunk 3 above)
Comment (SvetoslavKazandzhiev) [reply Carwash/Carwash/settings.py]: Done

**Review by strangerok1507 — APPROVED** [review]

**Pull request merged**